web/js/tasks.js

```diff
@@ -293,16 +293,25 @@ var TaskPanel = Ext.extend(Ext.Panel, {
                     reader:new Ext.data.XmlReader({record: 'project', id:'id' }, projectRecord),
                     remoteSort: false,
                     listeners: {
-                        'load': function () {
+                        'load': function (store) {
+                            var dummyRecord = new projectRecord({
+                                id: -1,                  // some invalid id
+                                description: "Load all projects",
+                                customerName: ""
+                            });
+
+                            store.add(dummyRecord);
+                            store.commitChanges();
+
                             //the value of projectComboBox has to be set after loading the data on this store
                             if ((this.findExact("id", this.parent.taskRecord.data['projectId']) == -1) &&
                                     (this.parent.taskRecord.data['projectId'] > 0)) {
                                 //no project with that id was found in this store
-                                if(this.baseParams.customerChanged) {
+                                if(this.baseParams.login) {
                                     //the project could not be found because the user
                                     //has just changed the client
```

Comment:
These code comments are not relevant. They belonged to the commit 679d38a, which fixed #144, but that doesn't apply anymore and I think this part of the code became useless. I understand you are reusing some useless code to implement the new feature, but please update the comments to explain it properly.